### ISSUE TITLE
Not to mark jupyter magic command as a problem

### ISSUE THREAD

**Issue by XJTLUmedia** [microsoft/pylance-release#3251]
Using environments: VSCode 1.70.0
Python 3.10.6
Pylance v2022.8.41

If I run code below

```python
from random import random
def estimate_pi(n=1e7) -> "area":
    """Estimate pi with monte carlo simulation.
    
    Arguments:
        n: number of simulations
    """
    in_circle = 0
    total = n
    
    while n != 0:
        prec_x = random()
        prec_y = random()
        if pow(prec_x, 2) + pow(prec_y, 2) <= 1:
            in_circle += 1 # inside the circle
        n -= 1
        
    return 4 * in_circle / total

%time estimate_pi()
```
![tt1](https://user-images.githubusercontent.com/102778070/187010973-25c0e0e4-4f53-4e10-9237-53e3049d5ace.png)
![ttt](https://user-images.githubusercontent.com/102778070/187010977-c05269c5-dd57-4abe-ad5b-c6d59d1f4f7c.png)

There is a problem on %time

Remove all problems notification on magic command of jupyter notebook. But not python

**Comment by debonte** [comment]
Triage: Magic parsing isn't completely broken -- there's no error if `%time estimate_pi()` is moved above the function.

**Comment by XJTLUmedia** [comment]
Yeah I see. It is not a big deal. But as this is kind of format suggestion to ask user put magic command on the top. It would be better to place it in warning rather than errors I think.

**Comment by debonte** [comment]
@XJTLUmedia, sorry for the confusion, I wasn't suggesting that you change your code. This was just a note to myself for discussing the issue with the team today during our triage meeting.

**Comment by debonte** [comment]
This issue has been fixed in prerelease version 2022.9.11, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022911-7-September-2022-PreRelease)
